.github/workflows/HcPkgPreReleaseGitops.yaml

```diff
@@ -0,0 +1,52 @@
+name: compare branches
+on:
+  workflow_call:
+   inputs:
+    main:
+     description:  "Main Branch"
+     required: true
+     type: string
+    releasing:
```

Comment:
this is the branch used to point to the released commit for the current stable/production hubs cloud package, I think naming it "releasing" maybe a bit confusing? something like "releaseTrackingBranch" or "HubsCloudBranch" maybe better?

Reply:
It would be better to call it releaseTrackingBranch because the name for that branch changes depending on the repo ex would be the reticulum branch. I am going to have to make these change in each repro and the script below.